sdk/cognitiveservices/cognitiveservices-qnamaker-runtime/README.md

```diff
@@ -26,14 +26,16 @@ npm install @azure/ms-rest-azure-js
 
 ##### Sample code
 
-```typescript
-import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
-import { QnAMakerClient } from "@azure/cognitiveservices-qnamaker";
+```javascript
+const {
+  QnAMakerRuntimeClient 
+} = require("@azure/cognitiveservices-qnamaker-runtime");
```

Comment:
Let's make this single line for better readability in samples. Am guessing your editor might have formatted it this way as the single line was too long?

```suggestion
const { QnAMakerRuntimeClient } = require("@azure/cognitiveservices-qnamaker-runtime");
```